scripts/admincommands.js

```diff
@@ -1,9 +1,8 @@
-exports.handleCommand = function(src, command, commandData, tar, channel) {
-    if (command === "memorydump") {
-        sys.sendMessage(src, sys.memoryDump(), channel);
-        return;
-    }
-    /*if (command == "togglerainbow") {
+/*global Config, exports, isSuperAdmin, kickbot, nameBans, nameWarns, nonFlashing, normalbot, require, sendChanHtmlAll, SESSION, sachannel, script, staffchannel, sys, tourchannel*/
+/*jshint shadow: true*/
+/*jslint continue: true, plusplus: true, sloppy: true, vars: true*/

```

Comment:
Weird lint/hint settings. We should agree on some before we start marking randomly. I'd remove for now